node/src/accountant/scanners/scan_schedulers.rs

```diff
@@ -445,12 +451,17 @@ mod tests {
                 Duration::from_secs(4),
             ),
         ];
-        let subject = NewPayableScanDynIntervalComputerReal::default();
+        let mut subject = make_subject();
```

Comment:
This is too vague, please keep the prefix `make` but use the struct name for suffix. For example, `make_scan_interval_computer`.